deployutils/apps/django/middleware.py

```diff
@@ -49,8 +49,23 @@
 class SessionMiddleware(BaseMiddleware):
 
     def process_request(self, request):
+        JWT_SESSION_STORE = \
+            'deployutils.apps.django.backends.jwt_session_store'
+        JWT_HEADER_NAME = 'HTTP_AUTHORIZATION'
```

Comment:
Should I move this to `settings.py`? I've added this to improve readability, but I doubt the values will be changed.